src/vsr/replica.zig

```diff
@@ -2122,6 +2147,8 @@ pub fn ReplicaType(
                 }
             }
 
+            prepare.commit_mins[message.header.replica] = message.header.commit_min;
```

Comment:
Assert the bounds on `message.header.commit_min`. The conservative low bound would be `op -| constants.journal_slot_count`, the upper bound is our own `commit_min`.